app/views/wiki/_header.html.erb

```diff
@@ -70,22 +70,23 @@
 
       <% if current_user && current_user.role == "admin" %>
         <li>
-          <%= link_to url_for(action: :delete,id: @node.id),method: :delete, data: {confirm: I18n.t('wiki.show.are_you_sure_delete', path: @node.path)} do %>
+          <a rel="tooltip" title="Delete this wiki page." data-placement="top" url_for(action: :delete,id: @node.id),method: :delete, data: {confirm: I18n.t('wiki.show.are_you_sure_delete', path: @node.path)} do>
```

Comment:
hi, @Apoorva-13 the error is here, for adding tooltip you have deleted the `link_to` and instead added `<a>` tag which is wrong as while making the transition you should also remove url_for, method, etc and add `href` and all in its place.Now,there are 2 solutions - you can add tooltip in link_to without changing it,take help from [this](https://stackoverflow.com/questions/37658396/adding-tooltip-in-link-to-tag-in-rail-application) or second solution is to add tooltip in `<li>` as `<li rel="tooltip" title="Delete this wiki page." data-placement="top">` which would also work.I hope it resolves the error.If you have any doubt regarding above 2 solutions, plz feel free to ask.Thanks.

Reply:
Thanks a lot for the help:)
would this work fine?
<%= link_to url_for(action: :delete,id: @node.id),method: :delete, data: {confirm: I18n.t('wiki.show.are_you_sure_delete', path: @node.path) },{:title => "Delete this wiki page"} do %>

Reply:
hi @Apoorva-13 I guess it would generate the error due to braces around title and also produces different style than other li items.You can use  this one 
> `add tooltip in <li> as <li rel="tooltip" title="Delete this wiki page." data-placement="top"> which would also work.`

Reply:
Thanks for the guidance!!
Would this work?
<li rel="tooltip" title="Delete this wiki page." data-placement="top">
            <%= link_to url_for(action: :delete,id: @node.id),method: :delete, data: {confirm: 
                     I18n.t('wiki.show.are_you_sure_delete', path: @node.path) } do %>
            <i class="fa fa-trash"></i>
            <span class="hidden-xs hidden-sm hidden-md hidden-lg"> <%= t('wiki.show.delete') %></span>
          </a>
          <% end %>
    </li>

Reply:
Thanks for the guidance!!

Reply:
Thanks a ton it passed all the checks:)